deploy/cloud/helm/platform/values.yaml

```
@@ -237,211 +357,245 @@ nats:
       registry:
 
-    # env var map, see nats.env for an example
+    # Environment variables for the reloader container
     env: {}
 
-    # all nats container volume mounts with the following prefixes
-    # will be mounted into the reloader container
+    # Volume mount prefixes from NATS container to share with reloader
+    # All NATS container volume mounts with these prefixes will be mounted into the reloader
     natsVolumeMountPrefixes:
     - /etc/
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)